packages/common/container-definitions/src/test/types/requestSignatures.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { IContainer } from "../../loader";
+
+export async function test(container: IContainer) {
```

Comment:
i recommend using `declare` for typing tests, and it will make it such that there is no runnable code emitted. I also wouldn't use the `spec.ts` extension, as to me it implies it is a runnable test.